test-reference/test_mnemoniccli_reference.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+#
+# BIP39 Mnemonic Phrase Generator and Verifier
+#
+# Secure Coding Principles and Practices (PA193)  https://is.muni.cz/course/fi/autumn2019/PA193?lang=en
+# Faculty of Informatics (FI)                     https://www.fi.muni.cz/index.html.en
+# Masaryk University (MU)                         https://www.muni.cz/en
+#
+# Team Slytherin: @sobuch, @lsolodkova, @mvondracek.
+#
+# 2019
+#
+import os
+import sys
+from binascii import unhexlify, hexlify
+from tempfile import TemporaryDirectory
+from typing import Optional
+
+from pa193mnemonicslytherin.mnemoniccli import Config, ExitCode
+from pa193mnemonicslytherin.test_mnemonic import get_random_valid_entropy_bytes, get_random_valid_mnemonic_phrase, \
+    get_random_valid_password
+from pa193mnemonicslytherin.test_mnemoniccli import TestMainBase
+
+try:
+    # noinspection PyPackageRequirements
+    from mnemonic import Mnemonic as TrezorMnemonic
+except ImportError as e:
+    print('{}\nPlease make sure `trezor/python-mnemonic` is installed. See `install.sh`.'.format(e), file=sys.stderr)
+    sys.exit(1)
+
+
+class TestMnemoniccliReference(TestMainBase):
+    """
+    Tests mnemoniccli tool against reference implementation of `trezor/python-mnemonic`.
+
+    https://github.com/trezor/python-mnemonic
+    """
+    _ENV_SUBTEST_COUNT = os.getenv('PA193MNEMONICSLYTHERIN_REFERENCE_SUBTEST_COUNT')  # type: Optional[str]
```

Comment:
could we increase the number of tests run in travis by defining this variable? since you mentioned before that travis can run up to 50 minutes, we might as well test it a bit more